client/src/mainPage/ideaSuggestions.js

```diff
@@ -1,57 +1,59 @@
 import React from 'react';
-import shuffle from 'shuffle-array';
 
-const suggestions = [
-  {option: "Romance"
-  },
-  {option: "Warzone"
-  },
-  {option: "Twist moment"
-  },
-  {option: "Betrayal"
-  },
-  {option: "Moment of truth"
-  },
-  {option: "Befriend"
-  },
-  {option: "Coming home"
-  },
-  {option: "Discover"
-  },
-  {option: "Ambush"
-  },
-  {option: "Murdered"
-  },
-  {option: "Suspense"
-  },
-  {option: "Crossroad"
-  },
-  {option: "Spying"
-  },
-  {option: "Following Clues"
-  },
-  {option: "Loss"
-  },
-  {option: "Bizarre"
-  },
-  {option: "Disease /Virus /Ravage"
-  },
-  {option: "Family Dispute"
-  },
-];
+class SuggestionsList extends React.Component
 
-setInterval(() => {
-  shuffle(suggestions);
+constructor(props){
+   super(props);
+   this.state = {
+      suggestions: [
+     {option: "Romance"},
+     {option: "Warzone"},
+     {option: "Twist moment"},
+     {option: "Betrayal"},
+     {option: "Moment of truth"},
+     {option: "Befriend"},
+     {option: "Coming home"},
+     {option: "Discover"},
+     {option: "Ambush"},
+     {option: "Murdered"},
+     {option: "Suspense"},
+     {option: "Crossroad"},
+     {option: "Spying"},
+     {option: "Following Clues"},
+     {option: "Loss"},
+     {option: "Bizarre"},
+     {option: "Disease /Virus /Ravage"},
+     {option: "Family Dispute"},
+     ],
+    shuffled: false,
+   }
+}
+
+shuffle = (array) => {
+  let arrayClone = JSON.parse(JSON.stringify(array))
+  arrayClone.sort(() => Math.random() - 0.5);
+  this.setState({
+     suggestions: arrayClone
+  })
+}
+
+componentWillMount(){
+    setInterval(() => {
+      this.shuffle(suggestions);
 }, 1000);
+}
 
-const SuggestionsList = () => (
-  <ul className='listOfSuggestions'>
+render(){
+   
+  return(
+    <ul className='listOfSuggestions'>
     {suggestions.map(idea => (
```

Comment:
Use `this.state` here , I kinda missed it

Reply:
Sorry...for what?